ui/src/ducks/inventory/index.js

```diff
@@ -1,17 +1,82 @@
+import axios from 'axios'
 import { createAction, handleActions } from 'redux-actions'
+import { openSuccess } from '../alerts'
 
 const actions = {
-  INVENTORY_GET_ALL: 'inventory/get_all',
+  INVENTORY_PENDING: 'inventory/pending',
+  INVENTORY_CREATE: 'inventory/create',
+  INVENTORY_UPDATE: 'inventory/update',
+  INVENTORY_DELETE: 'inventory/delete',
+  INVENTORY_REFRESH: 'inventory/refresh'
 }
 
 export let defaultState = {
-  all: []
+  all: [],
+  fetched: false,
 }
 
-export const findInventory = createAction(actions.INVENTORY_GET_ALL, () => {
```

Comment:
This function needs to be defined, it's trying to be called from `src/containers/InventoryLayout.js` and it can't find it.